### PR TITLE
Migrated CI from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by pluradj**
* Migrated backend tests and added OS coverage for Windows and Mac
* Created ModelZoo test report and synced results to project wiki

Signed-off-by: Jason Plurad <pluradj@us.ibm.com>

-----

The GitHub Actions workflows will run when this commit is pushed to a branch in this repository, so submitting the pull request won't execute the workflows. If you want to test this commit in this or your personal repo, you'd need to make a branch named `mz-something` and push this commit into that branch.

* CI test results: https://github.com/pluradj/onnx-tensorflow/runs/1474773812
* Model Zoo test report: https://github.com/pluradj/onnx-tensorflow/wiki/ModelZoo-Status-(branch=mz-gha)

### CODE REVIEW

File: test/test_modelzoo.py [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    if _CFG['verbose']:
+      print('Pulling {}{}'.format(file_path,
+                                  ' (dry_run)' if _CFG['dry_run'] else ''))
+    if not _CFG['dry_run']:
+      cmd_args = 'git lfs pull -I {} -X ""'.format(file_path)
+      subprocess.run(cmd_args,
+                     cwd=_CFG['models_dir'],
+                     shell=True,
+                     check=True,
+                     stdout=subprocess.DEVNULL)
```
Comment (chinhuang007) [inline test/test_modelzoo.py:121]: Do you expect a developer to run test_modelzoo.py in their own environment? If so, we should consider adding instructions on 1. getting git lfs installed, which is required for this python subprocess to work, 2. getting model zoo repo cloned into the current direct, which is also required when going with the default 'models' sub folder.

File: .travis.yml [hunk 2]
```diff
@@ -1,36 +0,0 @@
-os: linux
```
Comment (chinhuang007) [inline .travis.yml:1]: If we are no longer using travis, please make sure to update README and a couple python files where we still reference travis, to make the entire CI clean.

**Comment by pluradj** [comment]
Pushed a commit to update column headers
* CI test results: https://github.com/pluradj/onnx-tensorflow/runs/1495402623
* Model Zoo test report: https://github.com/pluradj/onnx-tensorflow/wiki/ModelZoo-Status-(branch=mz-gha)

**Review by chinhuang007 — APPROVED** [review]
Looks good!

**Pull request merged**